packages/paypal-commerce-integration/src/paypal-commerce-types.ts

```diff
@@ -573,3 +573,26 @@ export interface PayPalCreateOrderCardFieldsResponse {
     orderId: string;
     setupToken?: string;
 }
+
+export interface ProviderError extends Error {
+    errors?: ErrorElement[];
+    status?: string;
+    three_ds_result?: {
+        acs_url: unknown;
+        payer_auth_request: unknown;
+        merchant_data: unknown;
+        callback_url: unknown;
+    };
+}
+
+export interface ErrorElement {
+    code: string;
+    message: string;
+    provider_error?: {
+        code: string;
+    };
+}
+
+export function hasProviderErrors(error: unknown): error is ProviderError {
```

Comment:
This method should be mover into separate file in utils folder

Reply:
moved